R/SSplotSpawnrecruit.R

```diff
@@ -100,10 +100,11 @@ SSplotSpawnrecruit <-
   xlab <- labels[1]
   ylab <- labels[2]
   # check if spawning output rather than spawning biomass is plotted
-  if(replist$SpawnOutputUnits=='numbers'){ # quantity from test in SS_output
+  if (is.na(replist$SpawnOutputUnits) || 
```

Comment:
Defaulting to numbers rather than biomass when units are unavailable here differs from the approach I took in `SSplotTimeseries` which defaulted to biomass:
https://github.com/r4ss/r4ss/blob/development/R/SSplotTimeseries.R#L156-L157.
However, although biomass may be the more common units, the term "spawning output" can be inclusive of biomass as opposed to "spawning biomass" which does not work if the output is in some other unit. I will change `SSplotTimeseries` to match this better approach.

Reply:
That was my thinking, that if the units couldn't be found, the more general "spawning output" term could be used.